x-pack/plugins/observability_shared/public/components/page_template/page_template.tsx

```diff
@@ -29,13 +29,7 @@ import { NavNameWithBetaBadge } from './nav_name_with_beta_badge';
 
 export type WrappedPageTemplateProps = Pick<
   KibanaPageTemplateProps,
-  | 'children'
-  | 'data-test-subj'
-  | 'paddingSize'
```

Comment:
The `paddingSize` type was throwing errors in 5 or so template files. It's originally coming from `_EuiPageInnerProps` by way of `EuiPageTemplateProps>`. I can add it back and add a `paddingSize='SIZE'` declaration to the components. I'll look at EUI to see what the default should be.

Reply:
Updating this thread with context discussed off GitHub:

- This type change was causing issues specifically with the lazy/React.suspense usage of the page template (which tracks with some React 18 changes to suspense)
- Trevor spoke to the observability team and they're not using custom `paddingSize`s for their template anyway - @kdelemme approved removing this prop
- If `paddingSize` is still desired as a customizable prop, another alternative to removing it would be wrapping the `Pick<>` in a `Partial<>`, which resolves the type issue. If your team would rather see this change instead, please let us know.